index.md

```diff
@@ -36,6 +36,9 @@ img: ":home-heading.jpg"
 # don't forget that this is root index.html. If you disable this, there will be no index.html page to open
 #published: false
 ---
+<link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
There's no better way to add the fonts to all the pages? Adding it manually every time doesn't feel right.